### PR TITLE
#4674: Removed duplicated documentation from reference guide.

### LINKED ISSUE #4674: There are 2 pages of installation
There are currently two documentation pages about the installation of the bundle.

A first one in [Getting Started](https://sonata-project.org/bundles/admin/3-x/doc/getting_started/installation.html) which is complete and currently works.
A second one in [Reference Guide](https://sonata-project.org/bundles/admin/3-x/doc/reference/installation.html) which is not complete and outdated.

There is also a section called *Getting started* and a page in *Reference Guide* called [Getting started with SonataAdminBundle](https://sonata-project.org/bundles/admin/3-x/doc/reference/getting_started.html). The page *Getting started with SonataAdminBundle* should be removed.

### PR DESCRIPTION
**Pull request by gbalcewicz**
I am targeting this branch, because it targets the 3.x documentation.

Closes #4674

## Subject
There was outdated and duplicate documentation about getting started what is removed in this PR.

### CODE REVIEW

**Comment by greg0ire** [comment]
I removed the changelog, there is no need for one for docs (which implies pedantic) PRs

File: Resources/doc/reference/architecture.rst [hunk 1]
```diff
@@ -5,7 +5,7 @@ The architecture of the ``SonataAdminBundle`` is primarily inspired by the Djang
 Project, which is truly a great project. More information can be found at the
 `Django Project Website`_.
 
-If you followed the instructions on the :doc:`getting_started` page, you should by
+If you followed the instructions on the :doc:`../getting_started/creating_an_admin` page, you should by
```
Comment (greg0ire) [inline Resources/doc/reference/architecture.rst]: Here is what I would say normally : please comply with the sf docs standards:

> [Each line should break approximately after the first word that crosses the 72nd character (so most lines end up being 72-78 characters);](http://symfony.com/doc/current/contributing/documentation/standards.html)

But it does not seem to 72 chars here , so either adapt to what it currently is, or rewrap everything to 72 chars.

**Comment by jlamur** [comment]
Hello @gbalcewicz, thank you for your contribution.

It seems like you committed with an unknown email.
Please link the commit email with your github account or amend your commit to change its email.

**Comment by greg0ire** [comment]
> an unknown email

namely with the balcewicz.pl dn

**Comment by gbalcewicz** [comment]
@greg0ire Line what I have edited is not the longest one. All documentation files should be formatted but this is not scope of ticket this PR was done for.

**Comment by jlamur** [comment]
@greg0ire IMO, we need to setup *301 Moved Permanently* on removed pages.
Do you know how to achieve that?

**Comment by greg0ire** [comment]
@jlamur no idea

**Comment by jlamur** [comment]
I think that it must be done at the web server level.
Maybe @rande provided you a way to update it?

**Comment by greg0ire** [comment]
Nope but maybe @Soullivaneuh knows

**Comment by Soullivaneuh** [comment]
@jlamur If the doc is properly parsed, it should be automatically updated on the website when this PR will be merged.

**Comment by jlamur** [comment]
@Soullivaneuh I was talking about S.E.O. In particular, Google returns a page that will be deleted in this PR for the search query: "sonata admin install".

**Comment by Soullivaneuh** [comment]
@jlamur The sonata-project admin panel does not offer URL redirection option.

But in any case this is not a good idea because if two pages have the same final content, Google will remove one of them from the index.

It will be the same thing for not found page. It will be here for a time but will be deleted by Google if the 404 is not resolved.

So there is no problem AFAIK. At least, the 404 page could be manually deleted from webmaster tools but I don't have any access to it. cc @rande

**Comment by Soullivaneuh** [comment]
@jlamur Please list the needed redirection urls here. I'll take a look.

**Comment by jlamur** [comment]
- https://sonata-project.org/bundles/admin/{branch}/doc/reference/installation.html to https://sonata-project.org/bundles/admin/{branch}/doc/getting_started/installation.html
- https://sonata-project.org/bundles/admin/{branch}/doc/reference/getting_started.html to:
    - https://sonata-project.org/bundles/admin/{branch}/doc/getting_started/installation.html
    - or https://sonata-project.org/bundles/admin/{branch}/doc/getting_started/creating_an_admin.html

Not sure about the second one. Ideally, if there was something at https://sonata-project.org/bundles/admin/{branch}/doc/getting_started/ it should be redirect here.

**Comment by jlamur** [comment]
We will setup redirections later, thanks @gbalcewicz!

**Pull request merged**